### PR TITLE
EZP-29579 As an Editor, I want to see dates in preferred timezone

### PR DESCRIPTION
**Pull request by webhdx**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29579
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes?
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [ ] Coding standards (`$ composer fix-cs`)
- [ ] Ready for Code Review

### CODE REVIEW

**Comment by micszo** [comment]
Date is not changed on Notifications pop-up.
screen shot 2018-09-12 at 11 30 22

**Comment by micszo** [comment]
Date is not changed on UDW. \cc @sunpietro 
screen shot 2018-09-12 at 12 28 09

**Comment by micszo** [comment]
For the record, System Information / Composer doesn't seem to be covered.

**Comment by micszo** [comment]
Value in ezdatetime field is not changed.

File: src/bundle/Resources/public/js/scripts/admin.format.date.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+(function(moment) {
+    const formatEx = /[dDjlNSwzWFmMntLoYyaABgGhHisueIOPTZcrU]/g;
+    const formatMap = {
```
[the remark this answers is not in the document]
Comment (webhdx) [reply src/bundle/Resources/public/js/scripts/admin.format.date.js]: This provides compatibility with PHP's `DateTime` format.

**Comment by webhdx** [comment]
@mnocon Can you briefly check why Behat is failing here? Looks like Behat can't login to the UI for some reason.

**Comment by mnocon** [comment]
Sure, I'll look into it

**Comment by micszo** [comment]
On Dashboard in My Scheduled and All Scheduled dates are not changed.

**Pull request merged**